examples/pipeline/rag.py

```diff
@@ -54,7 +55,7 @@ class PromptTemplateComponent(Component):
     def __init__(self, prompt: PromptTemplate) -> None:
         self.prompt = prompt
 
-    async def run(self, query: str, context: list[str]) -> StringDataModel:
+    async def run(self, query: str, context: List[str]) -> StringDataModel:
```

Comment:
I'm not a fan of naming things after their type. Perhaps `RunOutput` or something else here?

Reply:
Done in https://github.com/neo4j/neo4j-genai-python/pull/119/commits/11a47b7c21c836a729cce2d26b7752efd1fc1484